modules/plugins/enigmail.js

```diff
@@ -173,6 +179,16 @@ if (hasEnigmail) {
     global.EnigmailConstants = EnigmailCommon;
   }
 
+  let isEnigmail2 = false;
```

Comment:
I think we should drop pre-enigmail 2 support, but I'm happy not to do that in this PR, we can do a follow-up. I've filed #1305 to remind us.

Reply:
I agree.
There are a lot of code for old Enigmail.
I'll remove old code at the next time.